### PR TITLE
Drawing View - Fix nullable, add doc to the public method

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Bugs Fixed ###

- Fixes #

### API Changes ###

<!-- List all API changes here (or just put None), example:

Added: 
 
- `string Class.Property { get; set; }`
- `void Class.Method();`

Changed:

 - `object Cell.OldPropertyName` => `object Cell.NewPropertyName`
 
If there is an entirely new API, then you can use a more verbose style:

```csharp
public static class NewClass {
    public static int SomeProperty { get; set; }
    public static void SomeMethod(string value);
}
``` -->

### Behavioral Changes ###

<!-- Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase. -->

### PR Checklist ###
<!-- Please check all the things you did here and double-check that you got it all, or state why you didn't do something -->
- [ ] Has a linked Issue, and the Issue has been `approved`
- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard
<!-- If at all possible, please update/add the documentation on the repo below. We would very much appreciate that. If you are unable to, please consider at least opening an issue on the repo below so we know that Docs still need to be adjusted/created. Thanks! <3 -->
- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

File: samples/XCT.Sample.WPF/MainWindow.xaml.cs
Comment (pictos) [inline samples/XCT.Sample.WPF/MainWindow.xaml.cs:17]: Sadly this a `no-go`. We can't have any `Init` call on XCT

**Review by pictos — CHANGES_REQUESTED** [review]
@VladislavAntonyuk Thanks for this! I'll not merge yet because of that small observation, I'll wait for your response. Thanks again

File: samples/XCT.Sample.WPF/MainWindow.xaml.cs
Comment (VladislavAntonyuk) [reply samples/XCT.Sample.WPF/MainWindow.xaml.cs:17]: Is it a bad practice or any violation in XCT?
As you can see it is used in PancakeView. Why can’t we do the same?

Comment (pictos) [reply samples/XCT.Sample.WPF/MainWindow.xaml.cs:17]: PacakeView isn't part of XCT.
So this is an agreement that we have on this project, that we want all devs to be able to use it without adding any other line of code. Like plug and play lib.

Comment (VladislavAntonyuk) [reply samples/XCT.Sample.WPF/MainWindow.xaml.cs:17]: I moved it to the Sample project. Init method is removed

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs:79]: if we receive a nullable canvas, should we trust that isn't null here? Does the base call check for null?

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.android.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.android.cs:62]: Should add some suggestion, like "please provide a bigger image" or "the image should have at least ${height} x ${width}" ?
```suggestion
				throw new Exception("Image is too small.");
```

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.gtk.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.gtk.cs:65]: same here

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.ios.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.ios.cs:56]: same here

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.macos.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.macos.cs:50]: same here

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.tizen.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.tizen.cs:61]: same here

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.uwp.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.uwp.cs:65]: same here

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.wpf.cs
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Service/DrawingViewService.wpf.cs:61]: here as well

**Review by pictos — COMMENTED** [review]
@VladislavAntonyuk I believe that this will be my last comments before merging this amazing work❣

**Review by pictos — APPROVED** [review]
Awesome!

**Pull request merged**
